### ISSUE TITLE
CUDNN_STATUS_ARCH_MISMATCH

### ISSUE THREAD

**Issue by AndsonYe** [rbgirshick/fast-rcnn#44]
Hi I have installed your fast-rcnn according to your instruction.  When I run the ./tools/demo.py I got a core dump. Bellow are the output:

F0817 22:53:22.694056 24820 cudnn_conv_layer.cpp:32] Check failed: status == CUDNN_STATUS_SUCCESS (6 vs. 0)  CUDNN_STATUS_ARCH_MISMATCH
*** Check failure stack trace: ***
Aborted (core dumped)

It seems I got a CUDNN_STATUS_ARCH_MISMATCH, I googled this one it seems it's due to my hardware. I am running this demo on a Nvidia Tesla M2090 GPU, whose CUDA compute capability is 2.0. How can I fix this problem?

Thanks.

**Comment by mdering** [comment]
cudnn needs CUDA compute 3.0, so you can't use cudnn with that card

**Comment by vikiboy** [comment]
@mdering I got the same with a Tesla K20x. I am using a custom dataset and train_faster_rcnn_alt_opt. Any idea on how to solve this issue ?

**Comment by mdering** [comment]
You might need to reboot if the drivers have updated recently

**Comment by vikiboy** [comment]
I haven't updated the drivers. I could restart it and check again, however, since it's a server, it would be helpful if there were any other options that doesn't entail restarting.

**Comment by mdering** [comment]
Right, I'm not sure what I did, but I think this is what fixed it for me.

**Comment by tongpinmo** [comment]
@mdering @AndsonYe my  CUDA compute capability is 6.1,but  I also encounter that problem,
Check failed: status == CUDNN_STATUS_SUCCESS (4 vs. 0)  CUDNN_STATUS_INTERNAL_ERROR
*** Check failure stack trace: ***
Aborted
 can you help me ?

**Comment by zhuozhongshuo** [comment]
I use "sudo apt-get install digits" to install digits, and finally solve this kinds of problems.
